client/model-config.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Role and ACL related models are in memory since they are used mostly for caching credentials on the client. They also wont be exposed on the server so cant be accessed over the remote connector.